python/cuml/cuml/experimental/hyperparams/HPO_demo.ipynb

```diff
@@ -447,7 +447,7 @@
     "        mode_str: User specifies what model it is to print the value\n",
     "    \"\"\"\n",
     "    y_pred = model.fit(X_train, y_train).predict(X_test)\n",
-    "    score = accuracy_score(y_pred, y_test.astype('float32'), convert_dtype=True)\n",
+    "    score = accuracy_score(y_pred, y_test.astype('float32'))\n",
```

Comment:
Wasn't sure if rendered notebooks should be updated, but it was a small diff to do so.